wfs/1.0.0/spatial-wfs-v1_0_0-common/pom.xml

```diff
@@ -38,5 +38,17 @@
             <groupId>org.codice.ddf.spatial</groupId>
             <artifactId>spatial-wfs-common</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.jvnet.ogc</groupId>
+            <artifactId>gml-v_3_2_1</artifactId>
+            <version>1.1.0</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.jvnet.ogc</groupId>
+            <artifactId>ogc-tools-gml32-jts</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Is there not a released version of this library?  I am not seeing a SNAPSHOT release in any public maven repo either.